ossm-release-notes/ossm-release-notes-feature-support-tables.adoc

```diff
@@ -1,12 +1,12 @@
 :_mod-docs_content-type: ASSEMBLY
-[id="ossm-release-notes-support-tables"]
+[id="ossm-release-notes-feature-support-tables"]
 = {SMProductShortName} {SMProductVersion} feature support tables
 include::_attributes/common-attributes.adoc[]
 :context: ossm-release-notes-support-tables
 
 toc::[]
 
-These tables provide guidance on supported versions and feature availability in {SMProduct} 3.
+These tables provide guidance on feature availability in {SMProduct} 3.
```

Comment:
```suggestion
{SMProductVersion} feature support tables provide guidance on feature availability in {SMProduct} 3.
```

Because this is a first sentence, suggest to replace "these tables" with "{SMProductVersion} feature support tables" for clarity.